docs/advanced_topics/formbuilder_routablepage_redirect.md

```diff
@@ -0,0 +1,50 @@
+# How to… use a redirect with Form builder to prevent double submission
```

Comment:
Not sure we need the `...` here.
```suggestion
 # How to use a redirect with Form builder to prevent double submission
```

Reply:
Thanks, I have mostly adopted this change.